### PR TITLE
Fix python pipeline for AzureEP without using root

### PR DESCRIPTION
**Pull request by RandySheriffH**
Fix python pipeline for AzureEP without using root, this is for 1.15.

### CODE REVIEW

File: tools/ci_build/github/linux/docker/inference/x64/python/cpu/scripts/install_centos.sh [hunk 1]
```diff
@@ -4,4 +4,14 @@ set -e
 os_major_version=$(cat /etc/redhat-release | tr -dc '0-9.'|cut -d \. -f1)
 
 echo "installing for os major version : $os_major_version"
-yum install -y which gdb redhat-lsb-core expat-devel tar unzip zlib-devel make libunwind bzip2 bzip2-devel
+yum install -y which gdb redhat-lsb-core expat-devel tar unzip zlib-devel make libunwind bzip2 bzip2-devel perl-IPC-Cmd python3 openssl-devel wget
```
Comment (snnn) [inline tools/ci_build/github/linux/docker/inference/x64/python/cpu/scripts/install_centos.sh:7]: Why do you need to install python3? The image already have all kinds of pythons under /opt . Installing an extra one could cause confusions.

Comment (RandySheriffH) [reply tools/ci_build/github/linux/docker/inference/x64/python/cpu/scripts/install_centos.sh]: Removed.

Comment (snnn) [reply tools/ci_build/github/linux/docker/inference/x64/python/cpu/scripts/install_centos.sh:7]: Then before you run this command, you should add one of the pythons to PATH

Comment (snnn) [reply tools/ci_build/github/linux/docker/inference/x64/python/cpu/scripts/install_centos.sh:7]: For example, 
```
export PATH=/opt/python/cp38-cp38/bin/python3.8:$PATH
```

Comment (RandySheriffH) [reply tools/ci_build/github/linux/docker/inference/x64/python/cpu/scripts/install_centos.sh]: replaced "export ...", now the package builds.

**Review by snnn — APPROVED** [review]
I don't think it will work. But if you insist on merging the PR, please feel free to do so.

**Pull request merged**